torch_geometric/contrib/flag/flag.py

```diff
@@ -2,6 +2,7 @@
 
 import torch
 
+from .flag_callback import FLAGCallback
```

Comment:
@Jamil I think you forgot to commit `FLAGCallback`

Reply:
oops! the `git add -u` muscle memory is too strong